tests/infrastructure/test_coverage_infrastructure.py

```
@@ -0,0 +1,3010 @@
+        from file_organizer.methodologies.para.rules.engine import EvaluationContext
+
+        ctx = EvaluationContext(file_path=Path("/home/user/doc.pdf"))
+        assert ctx.file_age_days is None
+
+    def test_evaluation_context_naive_datetime(self):
+        from datetime import UTC, datetime
+
+        from file_organizer.methodologies.para.rules.engine import EvaluationContext
+
+        ctx = EvaluationContext(
+            file_path=Path("/tmp/x.pdf"),
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/x.pdf"

(S108)